modules/gateway/gateway.go

```diff
@@ -81,16 +81,9 @@ import (
 // have previously been outbound peers, as it is less likely that those have
 // been manipulated.
 //
-// TODO: When peers connect to eachother, and when they add nodes to the node
-// list, there is no verification that the peers are running on the same Sia
-// network, something that will be problematic if we set up a large testnet.
-// It's already problematic, and currently only solved by using a different set
-// of bootstrap nodes. If there is any cross-polination (which an attacker
-// could do pretty easily), the gateways will not clean up over time, which
-// will degrade the quality of the flood network as the two networks will
-// continously flood eachother with irrelevant information. Additionally, there
-// is no public key exhcange, so communications cannot be effectively encrypted
-// or authenticated. The nodes must have some way to share keys.
+// TODO: There is no public key exhcange,
+// so communications cannot be effectively encrypted or authenticated.
+// The nodes must have some way to share keys.
```

Comment:
comment is incorrectly formatted. Most text editors have some shortcut or hotkey for highlighting a block of text and then snapping it to an 80 line word wrap (and will leave the `// ` parts of the comment intact and in the correct places). In vim, you can highlight the block of text in visual mode and then press `gq`